packages/streams/package.json

```diff
@@ -18,21 +18,8 @@
     "@types/chai": "^3.4.34",
```

Comment:
I very nearly asked you to clear _everything_ out of `devDependencies`, but I realize now that you're doing it because `$( yarn bin )` gives you the package's bin folder.

Can we move everything else out - leaving just `typescript` and `tslint`?  

Maybe your next task can be writing a really simple script that can live in the package root and run `$( yarn bin )/tsc --project packages/$PACKAGE_NAME/tsconfig.json` for every package in packages.  Then, we could move `lint`, `clean` and `build` out too.